arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/predef.kt

```diff
@@ -25,13 +24,13 @@ internal fun <A> Iterable<A>.deleteFirst(f: (A) -> Boolean): Pair<A, List<A>>? {
   return go(this, emptyList())
 }
 
-internal fun <A> Iterable<A>.uncons(): Pair<A, List<A>>? =
+fun <A> Iterable<A>.uncons(): Pair<A, List<A>>? =
```

Comment:
this function and the ones above should probably be moved to arrow-syntax?

Reply:
IMO, there is an entire set of `Iterable` operation missing from Kotlin Std or Arrow Syntax. This could also be isolated to Streams.

Reply:
Maybe is up for a discussion/ticket, my choices are: make them internal for our use or provide them with proper docs and all in arrow-syntax

Reply:
Moved to streams